communication/marketing-team/role-handbooks/storytellers.md

```diff
@@ -0,0 +1,32 @@
+# Storytellers 📖 Handbook
+
+## Overview
+
+It's no secret that governance and technical documentation can be kind of on the boring side. And that's ok - it's not there to put on a circus but provide valuable information and get you were you need to go.
+
+In this role, Storytellers are here for both! Storytellers will take us on journeys through words and visuals to help us craft...
+TODO add more here
+
+## Workflow ⚡️
+
+TODO  
+document a short workflow here
+1 should be for those who are comfortable going back and forth on a PR
+another option should include agreeing on a *final* version in a separate collabrative doc tool and the sig-docs-liasion or Editor will start the PR process
+
+## Minimum Skills and Experience
+- Have a passion for telling stories, technical writing, journalism, and painting pictures with words
+- Desire to be a Kubernetes Org Member and if not already, work your way towards membership
+- Comfortable with working with GitHub workflows or working in a collaborative doc tool with the Editor  
+
+## Expectations
+Anyone is welcome to contribute when they have time.
```

Comment:
I would like to add something to set expectation that this role is about someone signing up to complete an article. They're ultimately on point for getting that done. 

```suggestion
Anyone is welcome to contribute when they have time. The core expectation of storytelling is taking responsibility for getting the story from inception to published. 
```